tests/TestCase/ORM/MarshallerTest.php

```diff
@@ -221,6 +221,10 @@ public function testOneWithDatetimeField()
         $data['created'] = 1392387900;
         $result = $marshall->one($data, []);
         $this->assertSame($data['created'], $result->created->getTimestamp());
+
+        $data['created'] = '2014-02-14';
+        $result = $marshall->one($data, []);
+        $this->assertEquals('2014-02-14', $result->created->format('Y-m-d'));
```

Comment:
```suggestion
        $this->assertEquals(new Time('2014-02-14 00:00:00'), $result->created);
```

Reply:
this test will fail

Reply:
Then the format needs to be changed to re-create 00:00:00 time. `new DateTime('2014-02-14')` would create `2014-02-14 00:00:00`.

Reply:
I am sorry i don't understand. For me we have to edit masrhall for detect if it's 'Y-m-d' format and cast as 'Y-m-d 00:00:00' . rigth ?

Reply:
Yes, if only date part is passed then the time part should be always set to `00:00:00`, irrespective of current time.

Reply:
Shouldn't you be using the DateType if you only want to use the date portion or have a date column?